.doc_gen/cross-content/cross_RDSDataTracker_PHP_block.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "file://zonbook/docbookx.dtd"[
+    <!ENTITY % phrases-shared SYSTEM "file://AWSShared/common/phrases-shared.ent">
+    %phrases-shared;
+]>
+<block>
+
+    <para>
+        Shows how to use the &PHPlong; to create a web application that tracks work items in an &RDS; database and
+        emails reports by using &SESlong; (&SES;).
+        This example uses a front end built with React.js to interact with a RESTful PHP backend.
```

Comment:
It looks like the official name is just "React", according to what I see [here](https://reactjs.org/docs/getting-started.html).

Reply:
Going to leave it as is for clarity